lib/faker/tv_shows/suits.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Faker
+  class TvShows
+    class Suits < Base
+      flexible :suits
+
+      class << self
+        ##
+        # Produces a character from Suits
+        #
+        # @return [String]
+        #
+        # @example
+        #   Faker::TvShows::Suits.character #=> "Harvey Specter"
+        #
+        def character
+          fetch('suits.characters')
+        end
+
+        ##
+        # Produces a quote from Suits
+        #
+        # @return [String]
+        #
+        # @example
+        # Faker::TvShows::Suits.quote #=> "Don't play the odds, play the man."
+        #
```

Comment:
here too
missing `@faker.version next`
```suggestion
        #
        # @faker.version next`
```